lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -72,6 +80,21 @@ defmodule Membrane.H264.FFmpeg.Encoder do
                 spec: H264.profile_t() | nil,
                 default: nil
               ],
+              tune: [
+                description: """
+                Use in case of a specific input. For example, if your input is animation then use the animation tuning
+                or if you want to preserve grain in a film then use the grain tuning.
```

Comment:
I've found another source of this description from [`x264 --fullhelp`](https://superuser.com/questions/564402/explanation-of-x264-tune). Here's what I came up with: 
```suggestion
                Optionally tune the encoder settings for a particular type of source or situation.
                See [`x264` encoder's man page](https://manpages.ubuntu.com/manpages/trusty/man1/x264.1.html) for more info
```